src/amo/components/UserProfileEdit/styles.scss

```diff
@@ -13,6 +13,7 @@ $font-size-aside: 10px;
   }
 }
 
+.UserProfileEdit-authenticate,
 .UserProfileEdit-user-links {
```

Comment:
do we need this here? below you have the opposite..

Reply:
not actually you even need to declare `display:block` if you remove this

Reply:
I want to leverage the rest of the CSS for larger screens actually and I force `display: block` for all screen sizes. Does that make sense?

Reply:
yep got it 👍